drop/download_data.sh

```diff
@@ -14,6 +14,9 @@ else
     echo "Data directory not empty, is not updated"
 fi
 
+# move fds object
+mv Data/Output ./
```

Comment:
Why is this necessary?

Reply:
I added a fitted fds with ncbi chr styles to be downloaded with the demo. Then, I copy it to the Output folder resembling the path it would lie on if it had been generated bt DROP. Then, I test if gets correctly annotated with gene names even though it has a different chr style than the provided gtf file.

Reply:
I don't quite see where you copy the fds file, though. From this line it looks as if you're copying a whole directory?

Reply:
yes, I'm copying the whole Output folder, but it only contains the fds file under processed_data/aberrant_splicing ... In that way, it follows the same path and the rule to produce the results recognizes it

Reply:
This is only needed for testing or? Hence, I would not pollute the user's output folder with such things and have it rather in the testing logic. Basically, when you start testing you can copy it within the pytest code, but not by default.

Reply:
removed